packages/auth/src/api/account_management/mfa.ts

```diff
@@ -55,7 +57,10 @@ export interface StartPhoneMfaEnrollmentRequest {
   idToken: string;
   phoneEnrollmentInfo: {
     phoneNumber: string;
-    recaptchaToken: string;
+    recaptchaToken?: string;
+    captchaResponse?: string;
```

Comment:
I suggest add comments after each field like "reCAPTCHA v2", "reCAPTCHA Enterprise" for future reference.

Reply:
Done.